index.html

```diff
@@ -73,6 +76,196 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+
+        <!-- work card 2 -->
+        <div class="work_card">
+          <div class="work_preview">
+            <img src="./images/snapshoot-portfolio2.png" alt="Project 2" />
+          </div>
+          <div class="work_details">
+            <h1 class="work_title">Multi-Post Stories</h1>
+            <ul class="work_info">
+              <li class="w_s">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+            <p class="work_details_content">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+
+        <!-- work card 3 -->
+        <div class="work_card">
+          <div class="work_preview">
+            <img src="./images/snapshoot-portfolio3.png" alt="Project 3" />
+          </div>
+          <div class="work_details">
+            <h1 class="work_title">Tonic</h1>
+            <ul class="work_info">
+              <li class="w_s">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+            <p class="work_details_content">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+
+        <!-- work card 4-->
+        <div class="work_card">
+          <div class="work_preview">
+            <img src="./images/snapshoot-portfolio4.png" alt="Project 4" />
+          </div>
+          <div class="work_details">
+            <h1 class="work_title">Multi-Post Stories</h1>
+            <ul class="work_info">
+              <li class="w_s">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+            <p class="work_details_content">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+      </section>
+      <!-- end of work section-->
+
+      <section class="about">
+        <h1 class="headline_title">About Myself</h1>
```

Comment:
- [ ] KIndly consider using the correct header tag on your element. since only one h1 is allowed on the page kindly change this here to `h2...6`. and also consider placing it as it is on the provided template.
- [ ] Also you may consider using the same color and font-size.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198302032-664ee8cb-7b58-4de2-9281-35892b9eb223.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198301610-614bf7fe-eec2-4772-99c2-aaaa69d8aff8.png)